src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -860,7 +886,7 @@ std::shared_ptr<coreml::MLModelWrapper> drawing_classifier::export_to_coreml(
       {"features", features_string},
       {"max_iterations", read_state<flex_int>("max_iterations")},
       // TODO: Uncomment as part of #2524
-      // {"warm_start", read_state<flex_int>("warm_start")},
+      {"warm_start", read_state<flex_int>("warm_start")},
```

Comment:
I believe we don't expose opts unless we specify it at here.

Reply:
I'm not sure I follow, can you explain what you mean?

Reply:
@Jarvi-Izana This is for export-coreml metadata, not a user option.

Reply:
Isn't "warm_start" a string option?

Reply:
Good catch! Ty. This typo was the reason tests failed.